community-operators/federatorai/federatorai.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+          resources:
+          - alamedaservices
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterroles
+          verbs:
+          - create
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          verbs:
+          - list
+          - watch
+          - create
+        - apiGroups:
+          - ""
+          resources:
+          - persistentvolumeclaims
+          - services
+          - configmaps
+          - serviceaccounts
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - get
+          - create
+          - update
+          - delete
+        - apiGroups:
+          - federatorai.containers.ai
+          resources:
+          - alamedaservices/finalizers
+          verbs:
+          - '*'
+      - serviceAccountName: alameda-operator
+        rules:
+        - apiGroups:
+          - ''
+          resources:
+          - nodes
+          - replicationcontrollers
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - replicasets
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - apps.openshift.io
+          resources:
+          - deploymentconfigs
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - autoscaling.containers.ai
+          resources:
+          - alamedascalers
+          - alamedarecommendations
+          verbs:
+          - list
+          - watch
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - validatingwebhookconfigurations
+          verbs:
+          - create
+          - list
+          - update
+          - watch
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - create
+          - update
+      - serviceAccountName: admission-controller
+        rules:
+        - apiGroups:
+          - admissionregistration.k8s.io
+          resources:
+          - mutatingwebhookconfigurations
+          verbs:
+          - get
+          - create
+          - update
+      - serviceAccountName: alameda-datahub
+        rules:
+        - apiGroups:
+          - ''
+          resources:
+          - namespaces
+          verbs:
+          - get
+      permissions:
```

Comment:
The `permissions` section looks nearly identical to the `clusterPermissions`. These sections are used by OLM (operator-lifecycle-manager) to create the required RBAC for this operator, with `permissions` being used for `Roles` and `clusterPermissions` for `clusterRoles`. Does your operator need both?

Reply:
We have a refinement of these two parts. We try to ask as less rbac rights as possible in the clusterPermissions part. So resource may be duplicated but verbs should tell the different.

Reply:
We have a refinement commit of these two parts. We try to ask as less rbac rights as possible in the clusterPermissions part. So resource may be duplicated but verbs should tell the different.